xml/vk.xml

```diff
@@ -5129,6 +5135,10 @@ server.
             <param len="descriptorWriteCount" externsync="pDescriptorWrites[].dstSet">const <type>VkWriteDescriptorSet</type>* <name>pDescriptorWrites</name></param>
             <param optional="true"><type>uint32_t</type> <name>descriptorCopyCount</name></param>
             <param len="descriptorCopyCount" externsync="pDescriptorCopies[].dstSet">const <type>VkCopyDescriptorSet</type>* <name>pDescriptorCopies</name></param>
+            <implicitexternsyncparams>
+                <param>the sname:VkDescriptorPool that the pname:dstSet member of each pname:pDescriptorWrites element was allocated from</param>
+                <param>the sname:VkDescriptorPool that the pname:dstSet member of each pname:pDescriptorCopies element was allocated from</param>
+            </implicitexternsyncparams>
```

Comment:
I don't think these two are needed. The descriptor sets have already been allocated and these should just be writing to their allocations, which doesn't require synchronizing the pool.

And, practically speaking, if we made descriptor updates externsync on a common object it would make multithreading significantly harder.

Reply:
Agreed. It returns `void`, which probably means it cannot ask the pool for additional memory.